tracing-error/src/lib.rs

```diff
@@ -74,8 +71,59 @@
 //!     }
 //! }
 //! ```
-//! In the future, this crate may also provide its own `Error` types as well,
-//! for users who do not wish to use other error-handling libraries.
+//!
+//! This crate also provides the [`InstrumentResult`]/[`InstrumentError`] traits which can be used
+//! to wrap errors with a [`TracedError`] which bundles the inner error with a [`SpanTrace`].
+//!
+//! ```rust
+//! # use std::error::Error;
+//! use tracing_error::InstrumentResult;
```

Comment:
should we maybe just show importing the prelude in examples?
```suggestion
//! use tracing_error::prelude::*;
```